packages/core/src/plugins/plugins.types.ts

```diff
@@ -1,117 +1,123 @@
+  unknown = 'unknown',
 }
 
 export type SearchResultsArtist = {
-    id: string;
-    coverImage: string;
-    thumb: string;
-    name: string;
-    resourceUrl?: string;
-    source: SearchResultsSource;
-}
+  id: string;
+  coverImage: string;
+  thumb: string;
+  name: string;
+  resourceUrl?: string;
+  source: SearchResultsSource;
+};
 
 export type SearchResultsAlbum = {
-    id: string;
-    coverImage?: string;
-    thumb?: string;
-    title: string;
-    artist: string;
-    resourceUrl?: string;
-    type?: string;
-    source: SearchResultsSource;
-}
+  id: string;
+  coverImage?: string;
+  thumb?: string;
+  title: string;
+  artist: string;
+  resourceUrl?: string;
+  type?: string;
+  source: SearchResultsSource;
+};
 
 export type SearchResultsPodcast = {
-    id: string;
-    coverImage?: string;
-    thumb?: string;
-    title: string;
-    author: string;
-    resourceUrl?: string;
-    type?: string;
-    source: SearchResultsSource;
-}
+  id: string;
+  coverImage?: string;
+  thumb?: string;
+  title: string;
+  author: string;
+  resourceUrl?: string;
+  type?: string;
+  source: SearchResultsSource;
+};
 
 export type SearchResultsTrack = {
-    id: string;
-    title: string;
-    artist: string;
-    source: SearchResultsSource;
-}
+  id: string;
+  title: string;
+  artist: string;
+  source: SearchResultsSource;
+};
 
 export type ArtistTopTrack = {
-    artist: { name: string };
-    title: string;
-    thumb?: string;
-    playcount?: number;
-    listeners?: number;
-}
+  artist: { name: string };
+  title: string;
+  thumb?: string;
+  playcount?: number;
+  listeners?: number;
+};
 
 export type SimilarArtist = {
-    name: string;
-    thumbnail: string;
-}
+  name: string;
+  thumbnail: string;
+};
 
 export type ArtistDetails = {
-    id: string;
-    name: string;
-    description?: string;
-    tags?: string[];
-    onTour?: boolean;
-    coverImage?: string;
-    thumb?: string;
-    images?: string[];
-    topTracks: ArtistTopTrack[];
-    similar: SimilarArtist[];
-    source: SearchResultsSource;
-}
+  id: string;
+  name: string;
+  description?: string;
+  tags?: string[];
+  onTour?: boolean;
+  coverImage?: string;
+  thumb?: string;
+  images?: string[];
+  topTracks: ArtistTopTrack[];
+  similar: SimilarArtist[];
+  source: SearchResultsSource;
+};
 
 export type AlbumDetails = {
-    id: string;
-    artist: string;
-    title: string;
-    thumb?: string;
-    coverImage?: string;
-    images?: string[];
-    genres?: string[];
-    year?: string;
-    type?: AlbumType;
-    tracklist: Track[];
-    resourceUrl?: string;
+  id: string;
+  artist: string;
+  title: string;
+  thumb?: string;
+  coverImage?: string;
+  images?: string[];
+  genres?: string[];
+  year?: string;
+  type?: AlbumType;
+  tracklist: Track[];
+  resourceUrl?: string;
 };
 
 export type StreamQuery = {
-    artist: string;
-    track: string;
+  artist: string;
+  track: string;
+};
+
+export enum StreamFormats {
```

Comment:
Added stream formats, allows us to expand with other formats if / when required in future.